apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -641,4 +751,10 @@ private String getServiceName(@Nullable ClassLoader initiatingClassLoader) {
     public void resetServiceNameOverrides() {
         serviceNameByClassLoader.clear();
     }
+
+    public enum TracerState {
+        STOPPED,
```

Comment:
maybe describe a bit what are agent states and the main transitions between them.